Anabaena/index.html

```diff
@@ -0,0 +1,152 @@
+            the back of the cell (\(\leftarrow\)) to determine the correct
+            placement of cell division.
+          </li>
+          <li>
+            <b>Start Rule:</b> I start the simulation with a single short cell
+          </li>
+          <li><b>Growth Rule:</b>All cells grow linearly</li>
+          <li>
+            <b>Subsitution Rule 1:</b> Short cells mature into long cells when
+            they reach a certain length (2x in this simulation). The orientation
+            remains the same.
+          </li>
+          <li>
+            <b>Substitution Rule 2:</b> Long cells grow until they're long
+            enough to divide (3x length of short cell in this simulation). The
+            cells divide into two cells. The front of the old cell becomes a
+            short cell facing the same direction. The back of the old cell
+            becomes a long cell, but now is facing backwards.
+          </li>
+        </ul>
+        <p>
+          The substitution rules can be written more formally as the following
+          L-system. Notice that you need 4 rules instead of 2 to handle the
+          orientation of the cells.
+        </p>
+        <!-- prettier-ignore -->
+        <p>
+            $$
+            \newcommand{\la}[1]{\overset{\leftarrow}{#1}}
+            \newcommand{\ra}[1]{\overset{\rightarrow}{#1}}
+
+            \begin{align} 
+                \text{start} &= \ra{S} \\
+                \ra{S} &\to \ra{L} \\
+                \la{S} &\to \la{L} \\
+                \ra{L} &\to \la{L}\ra{S} \\
+                \la{L} &\to \la{S}\ra{L} \\
+            \end{align}$$
+        </p>
+        <p>
+          One interesting detail about this L-system is the overall growth
+          pattern follows the Fibonacci sequence. For example, let's look at the
+          lengths of the first several iterations:
+        </p>
+        <table>
+          <thead>
+            <tr>
+              <th>Cells</th>
+              <th>Length</th>
+            </tr>
+          </thead>
+          <tr>
+            <td>\(\ra{S}\) (start)</td>
+            <td>1</td>
+          </tr>
+          <tr>
+            <td>\(\ra{L}\)</td>
+            <td>1</td>
+          </tr>
+          <tr>
+            <td>\(\la{L}\ra{S}\)</td>
+            <td>2</td>
+          </tr>
+          <tr>
+            <td>\( \la{S}\ra{L} \ra{L} \)</td>
+            <td>3</td>
+          </tr>
+          <tr>
+            <td>\( \la{L} \la{L}\ra{S} \la{L}\ra{S} \)</td>
+            <td>5</td>
+          </tr>
+          <tr>
+            <td>\( \la{S}\ra{L} \la{S}\ra{L} \ra{L} \la{S}\ra{L} \ra{L} \)</td>
+            <td>8</td>
+          </tr>
+        </table>
+        <p>
+          Huh, while writing the above, I just noticed another pattern: In any
+          given iteration, all the L cells face the same way. Likewise, all the
+          S cells face the same way. However, the L cells and S cells face in
+          opposite directions.
```

Comment:
Make a note that this only works because we start from a single cell. If we started with `<--S S-->` now you have a mix.

Reply:
Eh, there's already so much detail here